tests/sentry/lang/native/test_symbolicator.py

```diff
@@ -265,3 +271,135 @@ def test_reverse_aliases(self, builtin_sources):
         reverse_aliases = symbolicator.reverse_aliases_map(builtin_sources)
         expected = {"sentry:ios-source": "sentry:ios", "sentry:tvos-source": "sentry:ios"}
         assert reverse_aliases == expected
+
+
+class TestIgnoredSourcesFiltering:
```

Comment:
fwiw we don't have a lot of pytest-style classes around, may be more consistent to start a new file entirely